eth2util/helpers.go

```diff
@@ -17,6 +18,37 @@ import (
 	"github.com/obolnetwork/charon/app/z"
 )
 
+func ValidateBeaconNodeHeaders(headers string) error {
```

Comment:
We used to add comments for each exported function, explaining its contract. See other exported functions for example.